docs/rfcs/0042-evaluation-for-updated-features.md

```diff
@@ -0,0 +1,134 @@
+# Evaluation for updated FeatureFlag
+
+
+## Proposal
+
+- This feature evaluates only those feature flags that have been updated since the last evaluation.
+  - Computational efficiency will be improved because the number of the feature flags to be evaluated will be reduced.
+- We can make the tag optional.
+  - Previously, specifying a tag was required when executing GetEvaluations to reduce the response size.
+  - But, with this proposal, the response size can be kept small, so specifying a tag can be made optional.
+
+
+
+
+## Implementation
+
+The SDK stores the`EvaluatedAt` value indicating the timestamp of the evaluation's execution.
+When GetEvaluations is called again, that timestamp is sent to the server as a request parameter.
+
+The server checks it against the updatedAt value of the feature flags, and only evaluates those that have been updated since the previous evaluation.
+
+As an exception, the following feature flags must be evaluated regardless of the timestamp value.
+- Feature flags that depend on the feature flags that need to be evaluated
+- (if user attributes have been updated) Feature flags with targeting rules
+
+If user attributes have been updated, the result of the evaluation may also change.
+Since the server has to know that update, the SDK sends the `IsUserAttributesChanged` flag to the server as a request parameter like the timestamp.
```

Comment:
I think that UserAttributesUpdated is better. What do you think?

Reply:
I think so too 👍 

SDK has the function named `updateUserAttributes`.
https://github.com/bucketeer-io/android-client-sdk/blob/73508d1642da3c6db052f66a7471032aad56c269/bucketeer/src/main/kotlin/io/bucketeer/sdk/android/BKTClient.kt#L28